ApollosProject/apollos-prototype
# [Onboarding] Adds ConnectedAskName Components to Onboarding

## DESCRIPTION

When proceeding through the onboarding process, a user will be able to enter their Name and it will be added to their profile.

### What does this PR do, or why is it needed?

This adds the `Query` and `Mutation` components to the already created `AskName` component and allows the information entered to be updated for the profile. This also adds formik and yup for form validation.

### What design trade-offs/decisions were made?

N.A.

### How do I test this PR?

You can launch the Onboarding Slides and after entering your name, you hit enter on the `lastName` and the query for updating the name will fire. You can then navigate to the connect page to see if your changes took effect.

### What automated tests did you write?

Updated tests for the `AskName` component and wrote tests for `AskNameConnected`

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
